CHANGELOG.md

```diff
@@ -26,6 +26,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   issue in previous version.
 - **CUMULUS-2583**
   - `QueueGranules` task now updates granule status to `queued` once it is added to the queue.
+- **CUMULUS-2695**
+  - Updates the example/cumulus-tf deployment to change
```

Comment:
Should be updated to reflect that the default int he user facing `tf-modules/cumulus` module was changed

Reply:
e5b03ca35f